internal/querycoordv2/checkers/controller.go

```diff
@@ -97,6 +97,7 @@ func (controller *CheckerController) Start(ctx context.Context) {
 				controller.check(ctx)
 
 			case <-controller.checkCh:
+			case <-controller.scheduler.IsEmpty():
```

Comment:
need `fallthrough` or the `checkCh` will not work

Reply:
right, I will reconstruct this logic